src/components/html/EcAppCard.html

```diff
@@ -1,25 +1,11 @@
-<a-card class="ec-app-card" @click="$emit('click')" hoverable>
-  <div slot="cover" class="text-center">
-    <img
-      v-if="iconUrl"
-      class="img-fluid"
-      :src="iconUrl"
-      :alt="app.title"
-    />
-    <a-icon v-else type="picture"/>
+<div class="card" @click="$emit('click')" hoverable style="height: 26rem">
+  <img v-if="iconUrl" class="card-img-top" :src="iconUrl" :alt="app.title" />
+  <div class="card-body">
+    <h5 style="white-space: nowrap; width: 100%; overflow: hidden; text-overflow: ellipsis;" class="card-text">
```

Comment:
```suggestion
    <h5 class="card-text text-truncate">
```

Acho que [`.text-truncate`](https://getbootstrap.com/docs/4.5/utilities/text/#text-wrapping-and-overflow) resolve aqui em vez desse inline style (evite)...